libkbfs/bare_root_metadata_test.go

```diff
@@ -42,6 +45,23 @@ func runTestOverMetadataVers(
 	}
 }
 
+func runTestsOverMetadataVers(t *testing.T, prefix string,
```

Comment:
As you clean up, please add a comment explaining what `prefix` is and what it implies exactly.